synapse/module_api/__init__.py

```diff
@@ -217,9 +216,6 @@ def register_spam_checker_callbacks(
         user_may_invite: Optional[USER_MAY_INVITE_CALLBACK] = None,
         user_may_send_3pid_invite: Optional[USER_MAY_SEND_3PID_INVITE_CALLBACK] = None,
         user_may_create_room: Optional[USER_MAY_CREATE_ROOM_CALLBACK] = None,
-        user_may_create_room_with_invites: Optional[
```

Comment:
Should we make the parameters to `register_spam_checker_callbacks` keyword only via the `*` flag?

Reply:
We could, I'd suggest opening a new issue for this.

Reply:
I put up a PR at #11975